templates.dist/fields.php

```diff
@@ -223,3 +223,94 @@ function renderRegionalAddition(string $inputName, array $selectedValue, bool $m
 		return $output;
 	}
 }
+
+
+if (!function_exists('renderParkingLot')) {
+	function renderParkingLot(array $parkingArray, string $language, string $locale = 'de_DE'): array
+	{
+		$messages = [];
+		foreach ($parkingArray as $key => $parking) {
+			if (!$parking['Count']) {
+				continue;
+			}
+			/* translators: 1: Name of parking lot, 2: Price */
+			$element = sprintf(__('%1$s at %2$s', 'onoffice'), getParkingName($key, $parking['Count']), formatPriceParking($parking['Price'], $language, $locale));
+			if (!empty($parking['MarketingType'])) {
+				$element .= ' (' . $parking['MarketingType'] . ')';
+			}
+			array_push($messages, $element);
+		}
+		return $messages;
+	}
+}
+
+if (!function_exists('formatPriceParking')) {
+	function formatPriceParking(string $str, string $language, string $locale): string
+	{
+		$digit = intval(substr(strrchr($str, "."), 1));
+		if (class_exists(NumberFormatter::class)) {
+			$format = new NumberFormatter($locale, NumberFormatter::CURRENCY);
+			if ($digit) {
+				$format->setAttribute(NumberFormatter::MAX_FRACTION_DIGITS, 2);
+			} else {
+				$format->setAttribute(NumberFormatter::MAX_FRACTION_DIGITS, 0);
+			}
+			return str_replace("\xc2\xa0", " ", $format->formatCurrency($str, "EUR"));
```

Comment:
Please use the currency from the estate (field `waehrung`). Each estate can potentially have a different currency.

Also update the other places where the currency is hard-coded to use the estate's currency.